lib/__tests__/registry-sync/ecr-mirror.test.ts

```diff
@@ -36,7 +37,7 @@ describe('EcrMirror', () => {
             Value: '123aass:password-key:AWSCURRENT',
           },
         ],
-        Image: 'public.ecr.aws/jsii/superchain:1-bullseye-slim-node18',
+        Image: DEFAULT_SUPERCHAIN_IMAGE,
```

Comment:
Are we really testing anything here, now? If we set `DEFAULT_SUPERCHAIN_IMAGE` to `"some-nonsensical-string"`, it will still pass.

Reply:
True. Copying the string there feels just as useless, but I can do it I suppose